UPDATING.md

```diff
@@ -0,0 +1,319 @@
+# Updating Academy Theme
```

Comment:
We shouldn't need a separate markdown file... let's document this once in our docs.layer5.io, link to it, and avoid duplication here.